modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,195 @@
+
+* {pipelines-shortname} adds the `buildah-ns` task. It improves container build security with user namespace isolation and remains compatible with the existing `buildah` task.
+
+* Pull requests that trigger a push event populate a dynamic `pull_request_number` variable automatically, allowing for a clear reference to the specific pull request associated with the push event and improving traceability.
+
+* {pipelines-shortname} Operator references the updated task image, ensuring the correct image version is used for pipeline tasks.
+
+.Pruner
+* The event-based `tektonpruner` is generally available. Enable it and configure it directly in the `TektonConfig` custom resource (CR).
+
+* The event-based pruner includes a telemetry service `ServiceMonitor` that tracks pruned `PipelineRuns` and `TaskRuns` on port `9090`. Metrics support monitoring, alerting, and debugging. This enables Prometheus-based monitoring of pruner metrics, improves observability, and requires no additional configuration.
+
+.{tekton-triggers}
+* You can opt out installing {tekton-triggers}, to reduce resource usage and provide finer control over component rollout in environments where triggers are managed separately or not required. This enhancement mirrors the optional nature of other {pipelines-shortname} components.
+
+.{tekton-results}
+* {tekton-results} supports PostgreSQL version 17, ensuring compatibility for Konflux transition from version 13. This upgrade enables a smooth migration of Konflux's Relational Database Service (RDS) instance of the database from version 13 to either version 16 or 17. 
+
+* {tekton-results} skips processing of already stored `PipelineRuns`, improving performance and reducing API server calls. This optimization enhances overall efficiency and responsiveness, aiming to reduce server load and conserve resources.
+
+* {tekton-results} supports a new flag for the `tekton-results-watcher` controller to disable live collection of Runs by setting the following flag in the `TektonConfig` CRD:
++
+[source, yaml]
+----
+#...
+options:
+    deployments:
+      tekton-results-watcher:
+        spec:
+          template:
+            spec:
+              containers:
+              - args:
+                - "--disable_storing_incomplete_runs=true"
+                name: watcher
+                resources: {}
+#...
+----
++
+If set to `true`, runs are not stored until they are complete. If `false`, runs are stored upon creation and continuously upserted until deletion (current default). Using `true` helps improve performance.
```

Comment:
continuously upserted whenever there's a change.

Probably, we can mention that this will become the default behaviour in coming releases. So you might need to explicitly set it to false.